src/utils.js

```diff
@@ -183,4 +183,25 @@ global.utils = {
     }
     return arrayParts;
   },
+
+  /**
+   * Given a direction, 1-8, increment it by some value and wrap back to 1-8
+   *
+   * @param {Number} direction
+   * @param {Number} change
+   * @return {Number}
+   */
+  changeDirection: function(direction, change) {
```

Comment:
`utils` was mainly used for methods to be executed in the screeps console, manual actions.

How about extending the `Number.prototype`?

Reply:
I wondered why there were so many things in utils that aren't used anywhere in the code.

Extending Number seems a bit weird to me, but might be the best alternate place. I would hope to eventually replace a lot of small operations like this with library functions, including _.random(1,8) as something.getRandomDirection(). Would that also go in Number, or elsewhere?

Reply:
extending js built-in objects, like `Number.prototype` is bad idea. I'd rather extend `RoomPosition` (not prototype. just like `RoomPosition.changeDirection`)

Reply:
extending RoomPosition sounds reasonable. There are some position functions that take directions. No other direction handling functions yet, though.

Reply:
K, sounds reasonable. Surely better than `global.utils`